signer/core/validation.go

```diff
@@ -128,7 +128,7 @@ func (v *Validator) validate(msgs *ValidationMessages, txargs *SendTxArgs, metho
 		if len(data) == 0 {
 			if txargs.Value.ToInt().Cmp(big.NewInt(0)) > 0 {
 				// Sending ether into black hole
-				return errors.New(`Tx will create contract with value but empty code!`)
+				return errors.New(`Tx will create contract with value but empty code! `)
```

Comment:
You accidentally added a space at the end of the error message. Also, why are backticks used here instead of plain quotes?